backend/api/src/DB/models.go

```diff
@@ -13,6 +13,7 @@ type User struct {
 	gorm.Model
 	Username string  `gorm:"uniqueIndex" form:"username" binding:"required" json:"username"`
 	Password string  `form:"password" binding:"required" json:"password"`
+	role     string  `form:"role" default:"student" json:"role"`
```

Comment:
```suggestion
	Role     string  `form:"role" default:"student" json:"role"`
```
Måste börja med stor bokstav när man sätter namn i en struct. Speciellt om den är exported för annars exporteras inte det fältet `role` som linter också skriver här.